src/applications/ds-playground/pages/V1V3Page.jsx

```diff
@@ -827,6 +828,103 @@ export default function V1V3Page() {
             </div>
           </div>
         </div>
+        {/* Process List Comparison */}
+        <div className="vads-l-row">
+          <h3>Process List</h3>
+          <div className="vads-l-col--12 vads-u-align-items--center vads-u-border-bottom--1px vads-u-border-color--primary medium-screen:vads-u-display--flex">
+            <div className="vads-l-col--12 small-screen:vads-l-col--6 vads-u-margin--1">
+              <VaProcessList>
+                <li>
+                  <h3>Check to be sure you can request a Board Appeal</h3>
+                  <p>
+                    You can request a Board Appeal up to 1 year from the date on
+                    your decision notice. (Exception: if you have a contested
+                    claim, you have only 60 days from the date on your decision
+                    notice to request a Board Appeal.)
+                  </p>
+                  <p>
+                    You can request a Board Appeal for these claim decisions:
+                  </p>
+                  <ul>
+                    <li>An initial claim</li>
+                    <li>A Supplemental Claim</li>
+                    <li>A Higher-Level Review</li>
+                  </ul>
+                  <p>
+                    <strong>Note: </strong>
+                    You can’t request a Board Appeal if you’ve already requested
+                    one for this same claim.
+                  </p>
+                </li>
+                <li>
+                  <h3>Gather your information</h3>
+                  <p>Here’s what you’ll need to apply:</p>
+                  <ul>
+                    <li>Your mailing address</li>
+                    <li>
+                      The VA decision date for each issue you’d like us to
+                      review (this is the date on the decision notice you got in
+                      the mail)
+                    </li>
+                  </ul>
+                </li>
+                <li>
+                  <h3>Start your request</h3>
+                  <p>
+                    We’ll take you through each step of the process. It should
+                    take about 30 minutes.
+                  </p>
+                </li>
+              </VaProcessList>
+            </div>
+
+            <div className="vads-l-col--12 small-screen:vads-l-col--6 vads-u-margin--1">
+              <VaProcessList uswds>
+                <li>
```

Comment:
v3 `VaProcessList` should be using `va-process-list-item` as children, not `li`

Reply:
Maybe we need to add a linting error to the `VaProcessList` component so this gets caught easier?